bitnami/flink/templates/taskmanager/statefulset.yaml

```diff
@@ -148,7 +148,7 @@ spec:
           {{- else if .Values.taskmanager.livenessProbe.enabled }}
           livenessProbe: {{- include "common.tplvalues.render" (dict "value" (omit .Values.taskmanager.livenessProbe "enabled") "context" $) | nindent 12 }}
             tcpSocket:
-              port: rpc
```

Comment:
What would be the reason to use `data` port for taskmanager, and `pgrep` for the jobmanager?

Reply:
The only port with a default value for `jobmanager` is rpc ([doc](https://nightlies.apache.org/flink/flink-docs-master/docs/deployment/config/)). The rest of port used by `jobmanager` don't seem crucial. We could check rcp.bind alternatively.  See the environment below:

```
      FLINK_MODE:                           jobmanager
      FLINK_CFG_JOBMANAGER_RPC_ADDRESS:     flink-jobmanager
      FLINK_CFG_JOBMANAGER_RPC_PORT:        6557
      FLINK_CFG_JOBMANAGER_RPC_BIND__PORT:  6555
      FLINK_CFG_JOBMANAGER_BIND__HOST:      0.0.0.0
      FLINK_CFG_REST_ADDRESS:               flink-jobmanager
      FLINK_CFG_REST_PORT:                  8078
      FLINK_CFG_REST_BIND__ADDRESS:         0.0.0.0
      FLINK_CFG_BLOB_SERVER_PORT:           9191
      BITNAMI_DEBUG:                        false
```

What do you think?